sklearn/metrics/classification.py

```diff
@@ -1353,11 +1354,21 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
     precision : float (if average is not None) or array of float, shape =\
         [n_unique_labels]
 
-    recall : float (if average is not None) or array of float, , shape =\
+    recall : float (if average is not None) or array of float, shape =\
         [n_unique_labels]
 
-    fbeta_score : float (if average is not None) or array of float, shape =\
-        [n_unique_labels]
+    fbeta_score :
+                if beta is a float :
```

Comment:
I'm not sure this will render correctly. I'm mobile or else I'd check the rendered documentation.